src/toaster.tsx

```diff
@@ -309,7 +309,10 @@ export const ToasterUI: React.FC<
   return (
     <ToastContext.Provider value={value}>
       {possiblePositions.map((currentPosition, positionIndex) => (
-        <Positioner position={currentPosition}>
+        <Positioner
+          position={currentPosition}
+          key={`positioner_${positionIndex}`}
```

Comment:
```suggestion
          key={currentPosition}
```

Reply:
Since `possiblePositions` is of type `ToastPosition` (`type ToastPosition = 'top-center' | 'bottom-center'`), we can use that instead of the index.